### DISCUSSION TITLE
Welcome to mineserver Discussions!

### DISCUSSION THREAD

**Discussion by Lygaen** [Lygaen/mineserver#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

PS : Ye I know it's the default welcome template message but you know...

**Comment by cmd420** [comment]
what are you implementing atm

**Comment by Lygaen** [comment]
Hi !
At this moment, I'm implementing Login logic so that players will be able to actually join the server.
I plan on making a console (like other minecraft server) next.
